src/transformation/context/context.ts

```diff
@@ -23,7 +22,7 @@ export class TransformationContext {
 
     public readonly options: CompilerOptions = this.program.getCompilerOptions();
     public readonly luaTarget = this.options.luaTarget ?? LuaTarget.LuaJIT;
-    public readonly isModule = isFileModule(this.sourceFile);
+    public readonly isModule = ts.isExternalModule(this.sourceFile);
```

Comment:
What if all imports are elided is a file still considered a module in that case?

Reply:
Yes